libs/openchallenges/challenge-search/src/lib/challenge-search.component.ts

```diff
@@ -133,6 +135,7 @@ export class ChallengeSearchComponent
   // set default values
   defaultPageNumber = 0;
   defaultPageSize = 24;
+  defaultDropdownOptionSize = 5;
```

Comment:
I didn't understand what this variable meant until I played with its value. I initially thought that it was related to the size of a UI component, though a size of 5 pixels wasn't making much sense. It's actually referring to a number of elements/options visible in the dropdown container.

I also noticed that this value control the number of elements/options displayed in the UI component at any time. The initial value of 5 makes the UI component feel a bit cramped. 10 looks like a better value. What do you think?

Reply:
see comment below